geetools/ee_profiler.py

```diff
@@ -79,17 +79,24 @@ def _to_dict(self, input: str) -> dict:
         # Initialize a dictionary to hold lists for each column
         headers = [anyascii(h.strip()) for h in lines[0].split()]
         result: dict = {header: [] for header in headers}
+        # functions to process/format each header
+        process = {
+            "EECU-s": lambda eecus: float(eecus) if eecus != "-" else None,
+            "CurrMem": lambda mem: self._memory(mem),  # Mem is a string to convert
```

Comment:
what is the point of displaying twice the same information ?

Reply:
you mean because I created lambda functions (in theory to be applied many times) to use them once? I know, it can be done differently, but I designed in such way because using indexing needs extra code due to different outputs; if it contains `CurrMem` the indices will change. To avoid hardcoding all that logic I just though it was cleaner to say _"this indicator process it this way when it appears"_ (regardless of its position).